### PR TITLE
Add styles folder + global.css #16

### LINKED ISSUE #16: Add styles folder + global.css
It's usefull to think websites in pieces: we'll realize there are fundamental elements that we can stylize globaly (like text) :)
Once styles folder + global.css are created, it would be great to implement global styles for elements like text and anchor tags

### PR DESCRIPTION
**Pull request by nsandoya**
- Create styles/global.css 
- Replaced Tailwind's default configuration for using it as the primary stylesheet for the project. This will ensure consistent styles across all pages. Future global styles could mix tailwind classes and css if needed
- Incorporate two new fonts to Tailwind (Montserrat and Gabarito)
- Add text global styles

Resolves #16

### CODE REVIEW

File: src/styles/globals.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  }
+  
+  @font-face {
+    font-family: 'gabarito';
+    src: url('../assets/fonts/Gabarito.ttf') format('truetype')
+  }
+
+@layer {
+    /* Global text styles */
+    h1, h2, h3{
```
Comment (IvanM9) [inline src/styles/globals.css:18]: Hi Nataly

I have reviewed the website and with this change, the color of the text "Bienvenido a Ecuador In Tech" in the Hero section seems to be dark :(

Comment (nsandoya) [reply src/styles/globals.css:18]: Hi @IvanM9 ,
Thank you! Bug is fixed now :)

**Review by IvanM9 — APPROVED** [review]

**Comment by davcortez** [comment]
I'm wondering why you chose those family fonts. I mean, what drives your decision? @nsandoya

**Comment by davcortez** [comment]
One more thing: when you write the commit message, try to use conventional commits or not, because you have different conventions. 
*I know it could be nitpicking, but it could help you a lot.*

**Comment by nsandoya** [comment]
Hi @davcortez ,

Thank you so much for your feedback about commit conventions,  and design 🌱

Regarding the **font styles**:
Based on your **initial design request** (shared one month ago), I worked around some core concepts to establish the website’s global style:

**Original request:**
_A color palette that conveys action and positivity..._

**Core design concepts:**
Discovery, connection, openness (a welcoming atmosphere for all backgrounds and perspectives), growth and impact.

Based on these ideas, I selected _Montserrat_ and _Gabarito_ font families. I applied these styles in the prototype I shared with you and Brayen last month, including ( [https://www.figma.com/board/1iIrtPGyUSQZHcSC0urQxu/ECinTech_est%C3%A9tica?node-id=0-1&t=PxlhON51BlnrpckM-1](FigmaJam) ) notes for reference, too. At that time, everything seemed okey regarding the font choices.

I completely understand if there wasn’t enough time to revisit these details or if you've changed your mind. 
If you agree, we could: **Retain one** of the proposed font styles, **revert** to Astro's default font style, or **explore** new font options.

Let me know what works best for you 🙂

**Review by davcortez — COMMENTED** [review]
LGTM

**Comment by davcortez** [comment]
@nsandoya Now you can merge this PR.
Thanks

**Pull request merged**